brave/brave-core
# Enable CheckJavaStyle presubmit check

Resolves https://github.com/brave/brave-browser/issues/30823

This PR does following:
1. Enables `CheckJavaStyle` checks for Brave java files
2. Introduces `brave-style-5.0.xml` file which is the copy of `chromium-style-5.0.xml`, where severity for all checks except RedundantImport/UnusedImports is changed error => warning
3. Fixes RedundantImport/UnusedImports style error in all files.

Remaining style errors are here attached: [presubmit_result.txt](https://github.com/brave/brave-core/files/11666460/presubmit_result.txt)

It is suggested to make review per commit, only 1st has important change; the rest are generated with script.



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
I. presubmit errors
1. Create PR with some java file changed
2. Add unused import
3. Commit change, run `npm run presubmit` - expected to see error about unused import

II. presubmit warnings 
1. Create PR with some java file changed
2. Add method 
```Java
private int GetWidth() {
    return 0;
}
```
3. Commit change, run `npm run presubmit` - expected to see warning `Method names should start with a lower case letter (e.g. getWidth())`
